app/src/main/java/org/jellyfin/androidtv/ui/preference/screen/PlaybackPreferencesScreen.kt

```diff
@@ -204,6 +203,66 @@ class PlaybackPreferencesScreen : OptionsFragment() {
 			}
 		}
 
+		category {
+			setTitle(R.string.pref_audio_directplay)
+
+			checkbox {
+				setTitle(R.string.lbl_codec_mpeg)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.mpegEnabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_pcm)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.pcmEnabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_aac)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.aacEnabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_dts)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.dtsEnabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_ac3)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.ac3Enabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_eac3)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.eac3Enabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_thd)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.thdEnabled)
+				depends { userPreferences[UserPreferences.videoPlayer] != PreferredVideoPlayer.EXTERNAL }
+			}
+
+			checkbox {
+				setTitle(R.string.lbl_codec_other)
+				setContent(R.string.desc_bitstream_generic)
+				bind(userPreferences, UserPreferences.otherAudioEnabled)
```

Comment:
As commented above, I think it makes sense to split out OPUS and FLAC as separate options.